site/content/docs/5.1/components/dropdowns.md

```diff
@@ -426,19 +426,17 @@ Trigger dropdown menus at the left of the elements by adding `.dropstart` to the
       <li><a class="dropdown-item" href="#">Separated link</a></li>
     </ul>
   </div>
-  <div class="btn-group" role="group">
-    <div class="btn-group dropstart" role="group">
-      <button type="button" class="btn btn-secondary dropdown-toggle dropdown-toggle-split" data-bs-toggle="dropdown" aria-expanded="false">
-        <span class="visually-hidden">Toggle Dropstart</span>
-      </button>
-      <ul class="dropdown-menu">
-        <li><a class="dropdown-item" href="#">Action</a></li>
-        <li><a class="dropdown-item" href="#">Another action</a></li>
-        <li><a class="dropdown-item" href="#">Something else here</a></li>
-        <li><hr class="dropdown-divider"></li>
-        <li><a class="dropdown-item" href="#">Separated link</a></li>
-      </ul>
-    </div>
+  <div class="btn-group dropstart" role="group">
```

Comment:
Do we have the right to change the structure of the component? Is a Boosted mod?
What will happent to projects who already have this type of dropdowns? Will they stop working properly?

Reply:
You're totally right, I'm not sure I can. Was wondering why we had a nested `.btn-group` for `.dropstart`, opened a PR on Bootstrap to see if it can be fixed this way. If not, I'll rollback changes here !